chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
```

Comment:
I think they are here: https://github.com/chaijs/chai/blob/2.3.0/lib/chai/interface/should.js#L90-L91
```javascript
    should['throw'] = should['Throw'];
    should.not['throw'] = should.not['Throw'];
```